src/opcode.rs

```diff
@@ -227,23 +227,36 @@ opcode! {
 }
 
 opcode! {
-    /// Read from pre-mapped buffers that have been previously registered with
+    /// Read from a file into a fixed buffer that has been previously registered with
     /// [`Submitter::register_buffers`](crate::Submitter::register_buffers).
     ///
+    /// The operation designated by this `opcode` will read data from a file at a specific offset
+    /// into a fixed buffer that has been registered into the `io-uring` instance via
+    /// [`Submitter::register_buffers`](crate::Submitter::register_buffers).
+    ///
+    /// The caller must ensure that the `buf_index` argument indexes into the list of buffers that
+    /// have been registered and that the buffer the index refers to has a base pointer of `buf` and
+    /// a length of `len`.
+    ///
     /// The return values match those documented in the `preadv2(2)` man pages.
+    ///
+    /// See the man pages for `io_uring_prep_read_fixed(3)` for more information.
```

Comment:
Referencing the previous comment, do you know which man page the documentation should direct to? I understand that `liburing` is supposed to be a higher level thing that can change, but unless I missed something, there is no man page for the specific opcode `IORING_OP_READ_FIXED` and the write equivalent.

As a side note, regardless of if `liburing` does decide to change how `io_uring_prep_read_fixed` works, shouldn't this crate follow that? If `opcode` in this crate was just an enum like it exists in C I think that would make sense to keep it separate from `liburing`, but because in this crate you have to actually build an opcode struct to create a submission queue entry, I think synchronizing with `liburing` is reasonable (In fact, I thought this crate _was_ a stand-in for `liburing`, so that's on me).